src/main/java/nl/tudelft/ti2206/group9/level/InternalTicker.java

```diff
@@ -34,6 +36,9 @@
 	/** The timer that schedules the TimerTask is stored in the instance. */
 	private final Timer timer;
 
+	/** Ticker needs to know the primaryStage to be able to send the death menu*/
+	private static Stage primaryStage = null;
```

Comment:
No it doesn't :P My checkstyle gives a warning "private field not used", therefore it should be removed :)